echopype/commongrid/mvbs.py

```diff
@@ -404,9 +407,37 @@ def bin_and_mean_2d(
     return final
 
 
+def _linear_transform(
```

Comment:
I think we can move this function to new module `echopype/utils/compute.py` since it's something that's used in multiple functions, like `compute_NASC`.